qw11qD/platform.py

```diff
@@ -12,64 +15,72 @@
 def create():
     """Lines A and D of QuantWare 21q-chip controlled with Quantum Machines.
 
-    Current status (check before using):
-    Line A (6 qubits) is NOT calibrated because signal is very noisy at low readout power,
-    possibly due to not using the TWPA pump.
-    Line D (5 qubits): calibrated with TWPA and latest status in:
+    Current status (check before using): Line A (6 qubits) is NOT
+    calibrated because signal is very noisy at low readout power,
+    possibly due to not using the TWPA pump. Line D (5 qubits):
+    calibrated with TWPA and latest status in:
     https://github.com/qiboteam/qibolab_platforms_qrc/pull/149
     """
     lo_map = {q: f"{q}/drive_lo" for q in ["D1", "D4", "D5"]}
     lo_map["D2"] = lo_map["D3"] = "D2D3/drive_lo"
 
     twpa_d = SGS100A(name="twpaD", address="192.168.0.33")
 
-    qubits = {}
-    for i in range(1, 6):
-        q = f"D{i}"
-        qubits[q] = Qubit(
-            name=q,
-            probe=IqChannel(
-                name=f"{q}/probe",
-                mixer=None,
-                lo=f"D/probe_lo",
-                acquisition=f"{q}/acquisition",
-            ),
-            acquisition=AcquireChannel(
-                name=f"{q}/acquisition", twpa_pump=twpa_d.name, probe=f"{q}/probe"
-            ),
-            drive=IqChannel(name=f"{q}/drive", mixer=None, lo=lo_map[q]),
-            flux=DcChannel(name=f"{q}/flux"),
+    qubits = {
+        f"D{i}": Qubit(
+            drive=f"D{i}/drive",
+            flux=f"D{i}/flux",
+            probe=f"D{i}/probe",
+            acquisition=f"D{i}/acquisition",
         )
+        for i in range(1, 6)
+    }
 
     # Connect logical channels to instrument channels (ports)
     # Readout
-    channels = [QmChannel(qubit.probe, "octave5", port=1) for qubit in qubits.values()]
+    channels = {
+        q.probe: IqChannel(device="octave5", path="1", mixer=None, lo="D/probe_lo")
+        for q in qubits.values()
+    }
     # Acquire
-    channels.extend(
-        QmChannel(qubit.acquisition, "octave5", port=1) for qubit in qubits.values()
-    )
+    channels |= {
+        q.acquisition: AcquireChannel(
+            device="octave5", path="1", twpa_pump=twpa_d.name, probe=q.probe
+        )
+        for q in qubits.values()
+    }
     # Drive
-    channels.extend(
-        [
-            QmChannel(qubits["D1"].drive, "octave5", port=2),
-            QmChannel(qubits["D2"].drive, "octave5", port=4),
-            QmChannel(qubits["D3"].drive, "octave5", port=5),
-            QmChannel(qubits["D4"].drive, "octave6", port=5),
-            QmChannel(qubits["D5"].drive, "octave6", port=3),
-        ]
-    )
+    channels |= {
+        qubits["D1"].drive: IqChannel(
+            device="octave5", path="2", mixer=None, lo=lo_map["D1"]
```

Comment:
Since we are creating the drive channels one by one anyway and not in a loop, we could even drop `lo_map` and hardcode the LO names here. `lo_map` is not used anywhere else anymore.

Reply:
Done in c97da13